### DISCUSSION TITLE
Linkerd with Wercker or other CI

### DISCUSSION THREAD

**Discussion by guimap** [linkerd/linkerd2#4747] [General]
Hi guys, i'm using the linkerd into my cluster, after following the getting started, it all worked well. But i have a doubt about what's the best way to add linkerd into a CI pipeline, how do u guys does to apply that modified yaml when u have an automated pipeline, like a wercker pipeline for example ?

Thanks guys :)

**Comment by WLun001** [comment]
any chance to install `linkerd CLI` to the CI pipeline?

**Comment by ihcsim** [comment]
@guimap I am not familiar with wercker, but you can use `linkerd [install|upgrade]` or `helm template` to output the Linkerd control plane YAML into version-control files, and let your CD tool deploys these resources to your cluster.
